images/win/windows2016.json

```diff
@@ -295,6 +295,7 @@
         },
         {
             "type": "powershell",
+            "skip_clean": true,
```

Comment:
Just out of curiosity — what it this for?

Reply:
Packer tries to perform clean up and it causes race condition at the moment when Finalize-VM script is completed already and Restart is not started yet (between tasks). This is an output for case without skip_line:

2021-12-20T21:24:52.2241859Z     vhd: Clean up various directories
2021-12-20T21:24:52.2259823Z     vhd: Removing C:\Recovery
2021-12-20T21:24:52.2263319Z     vhd: Removing C:\Windows\logs
2021-12-20T21:24:52.2269551Z     vhd: Removing C:\Windows\winsxs\manifestcache
2021-12-20T21:24:52.2274096Z     vhd: Removing C:\Windows\Temp
2021-12-20T21:24:52.2288819Z     vhd: Removing C:\Users\packer\AppData\Local\Temp
2021-12-20T21:26:49.7135157Z     vhd: Finalize-VM.ps1 - completed
2021-12-20T21:26:50.9987760Z ==> vhd: . : The term 'c:/Windows/Temp/packer-ps-env-vars-61c0b971-723c-873f-6a3f-cd380fddb097.ps1' is not recognized as the
2021-12-20T21:26:50.9991492Z ==> vhd: name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was
2021-12-20T21:26:51.0004532Z ==> vhd: included, verify that the path is correct and try again.
2021-12-20T21:26:51.0008843Z ==> vhd: At line:1 char:138
2021-12-20T21:26:51.0023378Z ==> vhd: + ... ontinue'};. c:/Windows/Temp/packer-ps-env-vars-61c0b971-723c-873f-6a3 ...
2021-12-20T21:26:51.0027038Z ==> vhd: +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-12-20T21:26:51.0032436Z ==> vhd:     + CategoryInfo          : ObjectNotFound: (c:/Windows/Temp...d380fddb097.ps1:String) [], CommandNotFoundException
2021-12-20T21:26:51.0044203Z ==> vhd:     + FullyQualifiedErrorId : CommandNotFoundException
2021-12-20T21:26:51.0047815Z ==> vhd:
2021-12-20T21:26:51.0054310Z ==> vhd: Restarting Machine
2021-12-20T21:26:51.1183721Z ==> vhd: Waiting for machine to restart...